### PR TITLE
add jwt + rbac guide

### PR DESCRIPTION
**Pull request by yuval-k**
image

### CODE REVIEW

File: gloo/docs/enterprise/authentication/jwt.md [hunk 1]
```diff
@@ -0,0 +1,258 @@
...
+for authentication with the credentials of the pod's service account.
+Inside the JWT are *claims* that provide idtentity information, and a signature to verification.
+
+To verify these JWT, the kubernetes api server is provided with a public key. We can use this public 
+key to perform JWT verification for kubernetes service accounts in Gloo.
+
+Let's see the claims for `svc-a` - the service account we just created:
+
+```shell
+kubectl exec test-pod cat /var/run/secrets/kubernetes.io/serviceaccount/token | cut -d. -f2 | base64 -d 2>/dev/null|jq .
```
Comment (mitchdraft) [inline gloo/docs/enterprise/authentication/jwt.md]:
```suggestion
kubectl exec test-pod cat /var/run/secrets/kubernetes.io/serviceaccount/token | cut -d. -f2 | base64 --decode 2>/dev/null|jq .
```

awesome command!
`-d` does not work on mac but `--decode` does

Comment (yuval-k) [reply gloo/docs/enterprise/authentication/jwt.md]: thanks!

**Comment by mitchdraft** [comment]
I'm getting odd behavior:
```
kubectl exec test-pod cat /var/run/secrets/kubernetes.io/serviceaccount/token | cut -d . -f2 | base64 --decode 2>/dev/null
{"iss":"kubernetes/serviceaccount","kubernetes.io/serviceaccount/namespace":"default","kubernetes.io/serviceaccount/secret.name":"svc-a-token-j5lwj","kubernetes.io/serviceaccount/service-account.name":"svc-a","kubernetes.io/serviceaccount/service-account.uid":"64d73077-8d71-11e9-8cf5-0800277caf56","sub":"system:serviceaccount:default:svc-a"
```

note the missing "}" at the end

even though this worked fine when run on a different secret:
```
kubectl exec -n glooshot glooshot-688955bc56-bpd85 cat /var/run/secrets/kubernetes.io/serviceaccount/token | cut -d. -f2 | base64 --decode 2>/dev/null | jq .
{
  "iss": "kubernetes/serviceaccount",
  "kubernetes.io/serviceaccount/namespace": "glooshot",
  "kubernetes.io/serviceaccount/secret.name": "glooshot-serviceaccount-token-vb69t",
  "kubernetes.io/serviceaccount/service-account.name": "glooshot-serviceaccount",
  "kubernetes.io/serviceaccount/service-account.uid": "c20acff6-8b94-11e9-97cd-42010a8e0023",
  "sub": "system:serviceaccount:glooshot:glooshot-serviceaccount"
}
```

**Comment by yuval-k** [comment]
as discussed on slack this is a base64 padding issue fixed in the last commit

**Pull request merged**